### ISSUE TITLE
Nodes that make you walk slower/faster

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest#7785]
##### Issue type
 - Feature request
 
##### Summary
I suggest a new feature to add to nodes: Walking speed modifier.

If this is added, players and other “walking” entities will have their walk speed multiplied by a fixed factor `f` as long they are standing on the node.
If the player jumps off such a node, they keep their modified speed until they hit solid ground again (which updates their speed).

Use cases:
* Swamp dirt which slows you down (maybe for `aotearoa` or `australia` mods)
* Wet sand
* Turbo tile to speed players up (maybe for some crazy kind of racing game?)

**Comment by SmallJoker** [comment]
For players this is doable and factors <1 were already requested in #1822. Entities however are fully controlled by the mods, so they would have to use callbacks to check whether the node below is such a magic node.
Use cases 1 & 2 are already covered by #1822 (i.e. duplicate), whereas 3 would probably be a side-effect of viscosity values below 1

**Comment by Wuzzy2** [comment]
No, this request is different.
This is for modifingy the player speed when walking ON a node.
#1822 is for setting the player speed when walking THROUGH a node. This is a small but important difference.

**Comment by DS-Minetest** [comment]
Another difference:
>If the player jumps off such a node, they keep their modified speed until they hit solid ground again

**Comment by sfan5** [comment]
This is already possible now and can be closed. @Wuzzy2

**Comment by Wuzzy2** [comment]
How?

**Comment by sfan5** [comment]
`move_resistance` - #10810

**Comment by Wuzzy2** [comment]
No, `move_resistance` is different. `move_resistance` is basically just the old liquid viscosity physics that were decoupled from the liquid code earlier. This affects the movement speed when *inside* the node (like in a liquid) and only really makes sense for non-walkable nodes. I should know this, as I have done this myself. ;-)

What I am requesting here is a move speed modifier when you *stand on* top of a walkable node, i.e. wet sticky dirt or mud to slow you down.

> If this is added, players and other “walking” entities will have their walk speed multiplied by a fixed factor f **as long they are standing on the node.**

**Comment by farooqkz** [comment]
Ah this could have good application in CTF. You place a bunch of these nodes to speed you up and as soon as you walk on them, they disappear quickly after. You get a good speedup to get your flag to the base. And of course it requires teamwork. E.g. other players placing these nodes for you.

Overall, I think it's a very good addition.
